client/components/happychat/composer.jsx

```diff
@@ -44,7 +44,11 @@ export const Composer = React.createClass( {
 				onMouseLeave={ this.scrollbleedUnlock }
 				>
 				<div className="happychat__message">
+					<label htmlFor="hc_message_textarea" style={{ display: "none"}}>
```

Comment:
Ah, I see what you're doing now. Yep, based on the description for the aria label, I think that's what we should do, but now that I understand what you were trying to do I could get behind the other version as well. My preference is for the visual appearance to be as close to the aria appearance as possible, though.